Error while checking build/O2/fullCI for f55513fc4f4ce14b22f4d9e2b17bbc879941d32b at 2024-07-30 10:43:
```
## sw/BUILD/o2checkcode-latest/log
--
========== List of errors found ==========
++ GRERR=0
++ grep -v clang-diagnostic-error error-log.txt
++ grep ' error:'
/sw/SOURCES/O2/13320-slc8_x86-64/0/Generators/src/FlowMapper.cxx:64:23: error: use nullptr [modernize-use-nullptr]
++ [[ 0 == 0 ]]
++ exit 1
--
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/13320/f55513fc4f4ce14b22f4d9e2b17bbc879941d32b/build_O2_fullCI/pretty.html).

--- a/Generators/src/FlowMapper.cxx
+++ b/Generators/src/FlowMapper.cxx
@@ -61,7 +61,7 @@ void FlowMapper::CreateLUT(TH1D* mhv2vsPt, TH1D* mhEccVsB)
   // std::make_unique<TH1F>("hSign", "Sign of electric charge;charge sign", 3, -1.5, 1.5);
 
   mhLUT = std::make_unique<TH3D>("mhLUT", "", nbinsB, binsB.data(), nbinsPt, binsPt.data(), nbinsPhi, binsPhi.data());
-  mhLUT->SetDirectory(0); // just in case context is incorrect
+  mhLUT->SetDirectory(nullptr); // just in case context is incorrect
 
   // loop over each centrality (b) bin
   for (int ic = 0; ic < nbinsB; ic++) {

